src/platform_impl/linux/x11/mod.rs

```diff
@@ -232,6 +249,10 @@ impl<T: 'static> EventLoop<T> {
         let (user_sender, user_channel) = std::sync::mpsc::channel();
         let (redraw_sender, redraw_channel) = std::sync::mpsc::channel();
 
+        let kb_state =
+            KbState::from_x11_xkb(unsafe { (xconn.xlib_xcb.XGetXCBConnection)(xconn.display) })
```

Comment:
I think we have a safe convenience method for getting the XCB connection.

Reply:
The only function like that I could find is defined on `x11::window::Window`, `KbState` is initialized independently of any window.